cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -443,7 +443,7 @@ class alignas(16) column_device_view : public detail::column_device_view_base {
   __device__ T element(size_type element_index) const noexcept
   {
     size_type index       = element_index + offset();  // account for this view's _offset
-    char const* d_strings = d_children[strings_column_view::chars_column_index].data<char>();
+    char const* d_strings = head<char>();
```

Comment:
This seems ok to me.
I was expecting something like `d_strings = static_cast<char const*>(_data);` since we have access the member `_data` here.

Reply:
replaced with `_data`